vars/buildExecute.groovy

```diff
@@ -102,7 +105,15 @@ void call(Map parameters = [:]) {
                     error "[${STEP_NAME}] buildTool not set and no dockerImage & dockerCommand provided."
                 }
         }
-
+        if (config.cnbBuild) {
+            if (!['npm', 'gradle', 'maven', 'mta', 'docker'].contains(config.buildTool)) {
+                throw new AbortException("ERROR - 'cnbBuild' does not support '${config.buildTool}' as a buildTool, consider using 'kanikoExecute' instead")
```

Comment:
Does this "kanikoExecute" comment make sense (if buildTool is not "kaniko")?
```suggestion
                throw new AbortException("ERROR - 'cnbBuild' does not support '${config.buildTool}' as a buildTool, consider using 'kanikoExecute' instead")
```

Reply:
That's a good point. We could remove that part of the error message so it's just: 

"ERROR - 'cnbBuild' does not support '${config.buildTool}' as a buildTool."